src/wasm-link.h

```diff
@@ -83,12 +83,12 @@ struct Section {
 
   union {
     /* CUSTOM section data */
-    SectionDataCustom data_custom;
+    SectionDataCustom custom;
     /* DATA section data */
     std::vector<DataSegment>* data_segments;
     /* MEMORY section data */
     Limits memory_limits;
-  };
+  } data;
```

Comment:
Kinda. I started getting a bug because this union wasn't initialized to zero. It seemed easiest to name it so I could do `WABT_ZERO_MEMORY(data)`